styles/mixins/_focus.scss

```diff
@@ -39,9 +39,10 @@ $-color-focus-outline: var.$color-focus-outline;
  */
 @mixin outline-on-keyboard-focus($inset: false) {
   @include outline($inset);
+
   // Selector for browsers using JS polyfill, which adds the "focus-visible"
   // class to a keyboard-focused element.
-  &:focus:not(.focus-visible) {
+  .js-focus-visible &:focus:not(.focus-visible) {
```

Comment:
`.js-focus-visible`. Is a top level class correct? If thats the case, for clarify I think braces help here as well as putting the & on a second line. 

```
.js-focus-visible {
  &:focus:not(.focus-visible) {
    ...
  }
}